hack/local-up-cluster.sh

```diff
@@ -15,6 +15,8 @@
 # limitations under the License.
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+VAR="eth0"
+HOST_IP=$(ifconfig $VAR | grep "inet" | head -n 1 | awk '{ print $2 }')
```

Comment:
I think we can use the same mechanism that uses the apiserver, that is using the IP from the interface with the default gateway.
There are multiple ways of obtaining this from the shell, per example, you can get the default gateway interface with 
```
ip route | grep default | grep -Po '(?<=(dev )).*(?= src| proto)'                                                                              main 
enp2s0
```

You can obtain the IP directly too if you query for an IP that is outside of the host, i.e.
```
$ ip route get 8.8.8.8
8.8.8.8 via 192.168.1.1 dev enp2s0 src 192.168.1.141 uid 1000 
    cache 

$ ip route get 1.1.1.1 | grep -Po '(?<=(src )).*(?= uid)'                                                                              main 
192.168.1.141
````

Whatever option we choose we should add some comments explaining it